spacy/errors.py

```diff
@@ -914,7 +914,9 @@ class Errors(metaclass=ErrorsWithCodes):
     E1034 = ("Node index {i} out of bounds ({length})")
     E1035 = ("Token index {i} out of bounds ({length})")
     E1036 = ("Cannot index into NoneNode")
-
+    E1037 = ("\"{attr}\" is not a valid value for attr.\n"
+             "Please consult https://spacy.io/api/token#attributes "
+             "for more information!")
```

Comment:
It is a problem that there is no good list of the attr values in the docs. We should probably add one, but in a separate PR. (We usually avoid "!" and also "please" for the most part.)

```suggestion
    E1037 = ("Invalid attribute value '{attr}'.")
```